src/parser/combinators.rs

```diff
@@ -28,6 +28,12 @@ macro_rules! seq {
         let ($input, $name, start) = $y($input)?;
         seq!($input start start $location => $($rest)*)
     });
+    // This rule specifically is the first bind that occurs, hence why
+    // the location here is forwarded to the rest of the macro as the start location.
```

Comment:
Lets update this comment, we can just say it is a mutable version of the previous rule